docs/source/about_map_batch.rst

```diff
@@ -0,0 +1,43 @@
+Batch mapping
```

Comment:
Should "mapping" be uppercase for consistency elsewhere? (Not sure what the convention is)

Reply:
I think the convention is lowercase, unless we refer to a class or something that requires an uppercase, or if you want to emphasize on specific words

Reply:
This actually brings up a good point on convention! I've been using the Google developer [style guide](https://developers.google.com/style/capitalization), and we should use sentence case for headings and titles. I just changed a couple of headings, so it should reflect that now. :)

For `mapping`, I think it is ok to leave it in lowercase since it is a general thing that isn't specific to HF.